packages/dlp/src/v2beta1/dlp_service_client.js

```diff
@@ -0,0 +1,513 @@
+ *   The first element of the array is an object representing [ListInfoTypesResponse]{@link ListInfoTypesResponse}.
+ *   The promise has a method named "cancel" which cancels the ongoing API call.
+ *
+ * @example
+ *
+ * var client = dlpV2beta1.dlpServiceClient();
+ * var category = '';
+ * var languageCode = '';
+ * var request = {
+ *     category: category,
+ *     languageCode: languageCode
+ * };
+ * client.listInfoTypes(request).then(function(responses) {
+ *     var response = responses[0];
+ *     // doThingsWith(response)
+ * }).catch(function(err) {
+ *     console.error(err);
+ * });
+ */
+DlpServiceClient.prototype.listInfoTypes = function(request, options, callback) {
+  if (options instanceof Function && callback === undefined) {
+    callback = options;
+    options = {};
+  }
+  if (options === undefined) {
+    options = {};
+  }
+
+  return this._listInfoTypes(request, options, callback);
+};
+
+/**
+ * Returns the list of root categories of sensitive information.
+ *
+ * @param {Object} request
+ *   The request object that will be sent.
+ * @param {string} request.languageCode
+ *   Optional language code for localized friendly category names.
+ *   If omitted or if localized strings are not available,
+ *   en-US strings will be returned.
+ * @param {Object=} options
+ *   Optional parameters. You can override the default settings for this call, e.g, timeout,
+ *   retries, paginations, etc. See [gax.CallOptions]{@link https://googleapis.github.io/gax-nodejs/global.html#CallOptions} for the details.
+ * @param {function(?Error, ?Object)=} callback
+ *   The function which will be called with the result of the API call.
+ *
+ *   The second parameter to the callback is an object representing [ListRootCategoriesResponse]{@link ListRootCategoriesResponse}.
+ * @return {Promise} - The promise which resolves to an array.
+ *   The first element of the array is an object representing [ListRootCategoriesResponse]{@link ListRootCategoriesResponse}.
+ *   The promise has a method named "cancel" which cancels the ongoing API call.
+ *
+ * @example
+ *
+ * var client = dlpV2beta1.dlpServiceClient();
+ * var languageCode = '';
+ * client.listRootCategories({languageCode: languageCode}).then(function(responses) {
+ *     var response = responses[0];
+ *     // doThingsWith(response)
+ * }).catch(function(err) {
+ *     console.error(err);
+ * });
+ */
+DlpServiceClient.prototype.listRootCategories = function(request, options, callback) {
+  if (options instanceof Function && callback === undefined) {
+    callback = options;
+    options = {};
+  }
+  if (options === undefined) {
+    options = {};
+  }
+
+  return this._listRootCategories(request, options, callback);
+};
+
+function DlpServiceClientBuilder(gaxGrpc) {
+  if (!(this instanceof DlpServiceClientBuilder)) {
+    return new DlpServiceClientBuilder(gaxGrpc);
+  }
+
+  var dlpServiceClient = require('grpc-google-privacy-dlp-v2beta1').client;
```

Comment:
https://github.com/googleapis/googleapis/pull/351 fixed this. It will be resolved with regeneration.